packages/theme/src/themes.ts

```diff
@@ -90,6 +94,10 @@ export const darkTheme = createTheme({
     primary: {
       main: blue[500],
     },
+    banner: {
+      info: '2E77D0',
```

Comment:
Why are these needed?

They don't have a `#` in front so I guess they're not gonna work anyway.